pallets/ethereum-transactions/src/tests/tests.rs

```diff
@@ -172,13 +172,13 @@ impl DefaultTransactionBuilder {
         return [3u8; 32]
     }
 
-    // The quorum is 2/3 of the validators plus 1
-    // Our validator list has 3 members, so the quorum must be 3
+    // The quorum is 1/3 of the validators
+    // Our validator list has 3 members, so the quorum must be 1
     // I chose not to implement the same logic of the quorum function
     // because that would defeat the point of the test being black box
     pub fn default_quorum() -> u32 {
```

Comment:
I think we should have a test for more values of validators. We should test at least for n=4, n=10, and maybe the transitions between sets of 3, eg, 6 and 7.